A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AirspeedIndicator.js

```diff
@@ -839,10 +884,33 @@ class Jet_PFD_AirspeedIndicator extends HTMLElement {
         }
     }
     updateTargetSpeeds(currentAirspeed) {
+        {
```

Comment:
Nitpicking, but any specific reason for the anonymous block?

Reply:
No Idea, I just stick with the original code, they did it the same way